server/service/src/backend_plugin/types/amc.rs

```diff
@@ -29,10 +29,6 @@ pub trait Trait: Send + Sync {
     fn call(&self, input: Input) -> PluginResult<Output>;
 }
 
-// average_monthly_consumption -> amc
```

Comment:
I copied this file for graphql_query.rs and realise these comment are not relevant anymore
* amc is fine here
* don't need macro it's pretty simple copy paste and replace